docs/content/en/docs-dev/user-guide/managing-piped/required-permissions.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Required Permissions"
+linkTitle: "Required Permissions"
+weight: 12
+description: >
+    This page describes what permissions are required for a Piped to deploy applications.
+---
+
+A Piped requires some permissions to deploy applications, depending on the platform.
+
+## For Amazon ECS
+
+You need the following IAM actions.
+
+- `ecs:CreateService`
+- `ecs:CreateTaskSet`
+- `ecs:DeleteTaskSet`
+- `ecs:DeregisterTaskDefinition`
+- `ecs:DescribeServices`
+- `ecs:DescribeTaskSets`
+- `ecs:RegisterTaskDefinition`
+- `ecs:RunTask`
+- `ecs:TagResource`
+- `ecs:UpdateService`
+- `ecs:UpdateServicePrimaryTaskSet`
+
+- `elasticloadbalancing:DescribeListeners`
+- `elasticloadbalancing:DescribeRules`
+- `elasticloadbalancing:DescribeTargetGroups`
+- `elasticloadbalancing:ModifyListener`
+- `elasticloadbalancing:ModifyRule`
```

Comment:
[ask] I am not familiar with AWS IAM, so I have a question. Is the IAM in AWS based on the API call? I'm concerned about whether other required permission exists. 👀

Reply:
Thanks, I forgot to list permissions that are required along with ECS/ELB/Lambda APIs.
I'll add them

Reply:
@ffjlabo 
I fixed and added the following action.

```json
         {
            "Effect": "Allow",
            "Action": [
                "iam:PassRole"
            ],
            "Resource": [
                "arn:aws:iam::<account-id>:role/<task-execution-role>",
                "arn:aws:iam::<account-id>:role/<task-role>"
            ]
        }
```